modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/controller/LayoutImportController.java

```diff
@@ -591,6 +594,23 @@ else if (ExportImportThreadLocal.isStagingInProcess() &&
 			if (group.isCompany() ^ companySourceGroup) {
 				throw new LARTypeException(LARTypeException.TYPE_COMPANY_GROUP);
 			}
+
+			StagingGroupHelper stagingGroupHelper =
+				StagingGroupHelperUtil.getStagingGroupHelper();
+
+			String groupFriendlyUrl = headerElement.attributeValue(
+				"group-friendly-url");
+
+			boolean companyGroup = stagingGroupHelper.isCompanyGroup(group);
+
+			if (((groupFriendlyUrl == null) && companyGroup) ||
```

Comment:
Is this first condition already covered by the latest one?

I think:

if `(groupFriendlyUrl == null) && companyGroup` is true, `!Objects.equals(groupFriendlyUrl, "/export-import") && companyGroup)` is true

Reply:
If the first condition is removed, could we simplify the if clause?